slash/resuming.py

```diff
@@ -115,7 +115,7 @@ def save_resume_state(session_result, collected_tests):
 
     for test_metadata in tests_with_no_results:
         test_to_resume = ResumeState(session_id=session_result.session.id, file_name=test_metadata.file_path,\
-                                     function_name=test_metadata.function_name, status=ResumeTestStatus.PLANNED)
+                                     function_name=test_metadata.address_in_file, status=ResumeTestStatus.PLANNED)
```

Comment:
Please rename `ResumeState` object's member from `function_name` to `address_in_file`

Reply:
In addition, please change the DB version (due to the above rename) by:
```
_DB_NAME = 'resume_state_v2.db'
```
(It is currently `v1`

Reply:
sure changed as per @ayalash comments